libs/net/dal/Migrations/1.0.16/Up/PostUp/00-NotificationRazorTemplate.sql

```diff
@@ -0,0 +1,80 @@
+DO $$
```

Comment:
I don't have a DB migration in yet, but the template originally provided was mapped to the option `Value`.  This is an incorrect implementation.